sdv/multi_table/base.py

```diff
@@ -292,13 +314,11 @@ def sample(self, scale=1.0, randomize_samples=False):
                 create more rows than the original data by a factor of ``scale``.
                 If ``scale`` is lower than ``1.0`` create fewer rows by the factor of ``scale``
                 than the original tables. Defaults to ``1.0``.
-            randomize_samples (bool):
-                Whether or not the data should change on each sample call. If ``True``
-                every time ``sample`` is called, different data will be generated, if ``False``
-                every time ``sample`` is called, the same data will be generated. Defaults to
-                ``False``.
         """
-        return self._sample(scale=scale, randomize_samples=randomize_samples)
+        with self._temporary_numpy_seed():
+            sampled_data = self._sample(scale=scale)
```

Comment:
any reason not to do this at the single table level?

Reply:
Yes, the `numpy.random.choice` is being used in the `hma.py` https://github.com/sdv-dev/SDV/blob/fdbafd4d8a9fffe00b5bfca169c44c07b48199c3/sdv/relational/hma.py#L470

With this we ensure that the primary keys and relations between tables is maintained the same.